linux/Platform.c

```diff
@@ -557,6 +559,27 @@ void Platform_getPressureStall(const char* file, bool some, double* ten, double*
    fclose(fd);
 }
 
+void Platform_getFileDescriptors(double* used, double* max) {
+   *used = NAN;
+   *max = 65536;
+
+   char procname[128];
+   xSnprintf(procname, sizeof(procname), PROCDIR "/sys/fs/file-nr");
```

Comment:
No need to concat at runtime, `PROCDIR "/sys/fs/file-nr"` is concatenated at compile time.